src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/QueueHealthDecider.java

```diff
@@ -17,31 +17,32 @@
 
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker.actions.Action;
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker.actions.ModifyQueueCapacityAction;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.grpc.Resource;
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.grpc.ResourceEnum;
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.flow_units.ResourceFlowUnit;
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.summaries.HotClusterSummary;
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.summaries.HotNodeSummary;
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.summaries.HotResourceSummary;
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.store.rca.cluster.NodeKey;
 import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.store.rca.cluster.QueueRejectionClusterRca;
-
 import java.util.ArrayList;
 import java.util.List;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 // This is a sample decider implementation to finalize decision maker interfaces.
 // TODO: 1. Read action priorities from a configurable yml
-// TODO: 2. Read current queue capacity from NodeConfigurationRca (PR #252)
 
 public class QueueHealthDecider extends Decider {
 
+  private static final Logger LOG = LogManager.getLogger(Decider.class);
   public static final String NAME = "queue_health";
 
   private QueueRejectionClusterRca queueRejectionRca;
   List<String> actionsByUserPriority = new ArrayList<>();
   private int counter = 0;
 
   public QueueHealthDecider(long evalIntervalSeconds, int decisionFrequency, QueueRejectionClusterRca queueRejectionClusterRca) {
```

Comment:
remove QueueRejectionClusterRca from the constructor as we don't consume anything directly from QueueRejectionClusterRca

Reply:
Don't we rely on `QueueRejectionClusterRca` flow units to check whether a thread pool queue is healthy/unhealthy - @rguo-aws ?

Reply:
Yes resolved this with ruizhen!